src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Mappers.kt

```diff
@@ -39,19 +39,20 @@ import net.rcarz.jiraclient.Project as JiraProject
 import net.rcarz.jiraclient.User as JiraUser
 import net.rcarz.jiraclient.Version as JiraVersion
 
-fun JiraAttachment.toDomain(jiraClient: JiraClient, issue: JiraIssue) = Attachment(
+fun JiraAttachment.toDomain(jiraClient: JiraClient, issue: JiraIssue, config: Config) = Attachment(
```

Comment:
Maybe we could also in the future remove more hardcoded values (e.g. Mojira user group names) and add them to the config. Then we could benefit from the config being available here as well.
But you are right, it is a bit annoying that the config has to be passed around to so much functions.

Reply:
Something I was thinking is saving the config to a public car somewhere and just calling that statically instead of passing it around

Less functional, but prettier

Reply:
We could also make `Mappers` a class which has `JiraClient` and `Config` as properties, similar to your [`MapFromJira`](https://github.com/mojira/arisa-kt/blob/9a354f3c2e264cc7eb01f3fcb4174921b0ac5b6e/src/main/kotlin/io/github/mojira/arisa/infrastructure/newjira/MapFromJira.kt) class from #720, or my (incomplete) [changes on the `marcono1234/refactor` branch](https://github.com/mojira/arisa-kt/blob/marcono1234/refactor/src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Mapper.kt).
This can greatly reduce code duplication.